lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -53,6 +52,19 @@
     }
   }
 
+  // TYPE: inline
+  .msg-inline {
+    // @extend %t-copy-sub1;
+    display: inline-block;
+    margin: 0 0 0 $baseline;
```

Comment:
Do you need to reset the other three margin properties here or would ``margin-left`` be better suited?